x/uibc/quota/keeper/grpc_query_test.go

```diff
@@ -62,8 +71,9 @@ func TestGRPCQueryOutflows(t *testing.T) {
 			resp, err := client.Outflows(ctx, &tc.req)
 			if tc.errMsg == "" {
 				assert.NilError(t, err)
-				assert.DeepEqual(t, sdk.NewDec(0), resp.Amount)
+				assert.DeepEqual(t, sdk.NewDec(tc.outflowAmount), resp.Amount)
 			} else {
+				// query outflows never returns error if there is no denom present it will return 0
```

Comment:
1. remove else the comment comment about the err  just above `client.Outflows` call
2. remove `errMsg` from the struct